Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.Controls.WebView/WebBrowserCompatibilityAdapter.cs

```diff
@@ -0,0 +1,82 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Windows.Forms;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
+
+namespace Microsoft.Toolkit.Forms.UI.Controls
+{
+    internal sealed class WebBrowserCompatibilityAdapter : WebBaseCompatibilityAdapter
+    {
+        private WebBrowser _browser = new WebBrowser();
+
+        private void OnBrowserNavigated(object sender, WebBrowserNavigatedEventArgs e)
+        {
+            NavigationCompleted?.Invoke(sender, e);
+        }
+
+        private void OnBrowserNavigating(object sender, WebBrowserNavigatingEventArgs e)
+        {
+            NavigationStarting?.Invoke(sender, e);
+            ContentLoading?.Invoke(sender, e);
+        }
+
+        public override Uri Source { get => _browser.Url; set => _browser.Url = value; }
+
+        public override bool CanGoBack => _browser.CanGoBack;
+
+        public override bool CanGoForward => _browser.CanGoForward;
+
+        public override Control View => _browser;
+
+        public override event EventHandler<WebViewControlNavigationStartingEventArgs> NavigationStarting;
+
+        public override event EventHandler<WebViewControlContentLoadingEventArgs> ContentLoading;
+
+        public override event EventHandler<WebViewControlNavigationCompletedEventArgs> NavigationCompleted;
+
+        public override bool GoBack()
+        {
+            _browser.GoBack();
+            return true;
```

Comment:
This doesn't seem like the right thing to do as applications may need to know whether the navigation state has successfully moved to the previous document in the stack

Reply:
Since it is indeterminate between the two implementations, the interface should be `void GoBack()` and `void GoForward()`, or use nullable return e.g. `bool? GoBack()` and `bool? GoForward()` which will return true in the case it is known; otherwise, false. I'm in favor of the `void` implementation as it keeps the implementation simple.

Reply:
returning bool values from previous call on _browser